src/test/java/seedu/simplykitchen/logic/commands/CommandTestUtil.java

```diff
@@ -54,16 +54,22 @@
 
     public static final String INVALID_DESCRIPTION_DESC = " " + PREFIX_DESCRIPTION
             + "D&nut"; // '&' not allowed in descriptions
+    public static final String INVALID_DESCRIPTION_TOO_LONG = " " + PREFIX_DESCRIPTION
```

Comment:
Perhaps to standardise with the other test strings and for clarity, this can be INVALID_DESCRIPTION_LENGTH (or something along the lines of this)? Similar for the other added strings too.

Reply:
Fixed. Thanks.